src/App.module.scss

```diff
@@ -80,6 +80,13 @@
     }
   }
 
+  .sidebarButton {
+    background-color: white;
```

Comment:
Use the css variable var(--header) for the background color of the header button to take in the darkmode colors and var(--contrast) for the font color